service.py

```diff
@@ -410,6 +412,9 @@ def onPlayBackStarted(self):
 
                 data = {'action': 'started'}
 
+                if (kodiUtilities.getSettingAsBool('scrobble_mythtv_pvr')):
```

Comment:
This one is just a debug message - if you don't put it this far up in the branch tree, it's impossible to tell if a failure to scrobble is due to incorrect incoming data, or because the option is disabled by the user.